src/test/kotlin/com/cjbooms/fabrikt/generators/ModelGeneratorTest.kt

```diff
@@ -121,14 +122,18 @@ class ModelGeneratorTest {
             ?.let(::readFolder)
             ?: emptyMap()
         tempFolderContents.forEach {
-            if (expectedModels.containsKey(it.key)) {
-                assertThat((it.value))
-                    .describedAs("expected model '${it.key}' does not match the given value.")
-                    .isEqualTo(expectedModels[it.key])
+            if (expectedModels.contains(it.key)) {
+                assertThatGenerated(it.value)
```

Comment:
this will allow the auto-correction to be turned on if there are mass file changes required in future.